client/src/components/teams/InviteMembers.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from "react";
+
+import {
+  Button,
+  Grid,
+  InputLabel,
+  TextField,
+  makeStyles,
+} from "@material-ui/core";
+
+const InviteMembers = ({ setTeamMembers, teamMembers }) => {
+  const classes = useStyles();
+
+  const [email, setEmail] = useState("");
+  const [emailValid, setEmailValid] = useState(false);
+  const handleEmailChange = (event) => {
+    event.preventDefault();
+    //validation could probably be done better
```

Comment:
if you can think of a better way you should do it 😅 
This looks fine for now. If we get to it we can make this an autocomplete box that queries the current users